aten/src/ATen/core/boxing/impl/boxing.h

```diff
@@ -205,17 +153,21 @@ struct BoxedKernelWrapper<
     void
   >
 > {
+  static torch::jit::Stack boxArgs(Args... args) {
+    // TODO Reuse stack vector instead of allocating?
+    torch::jit::Stack stack;
+    stack.reserve(sizeof...(Args));
```

Comment:
Should this reserve be called inside torch::jit::push? Maybe other call sites would benefit from it too.

Reply:
True, I can't think of a downside... OTOH I'm not sure I want to colocate that change with the other stuff in this PR 😁